.changelog/1496.txt

```diff
@@ -0,0 +1,4 @@
+```release-note:enhancement
+access_application: Add support for allow_authenticate_via_warp
+access_organization: Add support for allow_authenticate_via_warp and warp_auth_session_duration
+```
```

Comment:
```suggestion
```release-note:enhancement
access_application: Add support for allow_authenticate_via_warp
```

```release-note:enhancement
access_organization: Add support for `allow_authenticate_via_warp` and `warp_auth_session_duration`
```
```